scripts/west_commands/build.py

```diff
@@ -150,6 +150,15 @@ def do_add_parser(self, parser_adder):
                            Do not use this option with manually specified
                            -DSHIELD... cmake arguments: the results are
                            undefined''')
+        group.add_argument('-k', '--kconfig-extra', help='''Additional Kconfig
+                           file to pass to cmake via EXTRA_CONF_FILE. Unlike
+                           specifying -DEXTRA_CONF_FILE=FILE as additional
+                           cmake arguments via cmake_opt, this does not imply
+                           '-c', i.e. cmake will not be forcibly re-run,
+                           unless there is some other reason to, e.g. FILE
+                           was modified. This can be used to implement
+                           multiple build configurations
+                           (e.g. Debug/Release).''')
```

Comment:
Initial thought here is that you should allow to pass multiple ``--kconfig-extra` files if we were to to allow this.